static/styles/left_sidebar.css

```diff
@@ -235,24 +235,14 @@ li.show-more-topics {
         margin-left: 0;
         margin-bottom: 0;
 
-        .active_private_messages_section {
-            background-color: hsl(202, 56%, 91%);
-            border-radius: 4px;
-
-            #more_private_messages_sidebar_title {
-                font-weight: 600;
-            }
-        }
-
```

Comment:
this `. active_private_messages_section` is a class which gets added to division with `#private_messages_section`  ID itself, hence it should be present outside this {} to get implemented correctly.